index.html

```diff
@@ -67,5 +67,29 @@ <h2>Tools</h2>
       </div>
     </main>
     <script src="./script.js"></script>
+
+    <!-- chatbot implementation-->
+    <span> 
+      <section>
+          <a class="float" id="float">
+              <img src="chatrobot.jpg" id="roboImg">
+          </a>
+          <div class="popup" style="width:500px;height:500px;">
+              <div class="textarea">
+                  <div class="roboside">
+                      <img src="chatrobot.jpg" class="avatar" alt="">

```

Comment:
Can you please change the robot pic with the logo of student portal in the chat bot?

Reply:
Ohkk, I will do above listed changes by evening! Hope it works. Can you provide the questions chart we actually want the chatbot to answer, so I can change it as well within JS.

Reply:
I have successfully done the changes, plz do review now!